Sources/SwiftGen/Config/Config+Example.swift

```diff
@@ -0,0 +1,81 @@
+//
+// SwiftGen
+// Copyright © 2019 SwiftGen
+// MIT Licence
+//
+
+extension Config {
+  static func example(versionForDocLink: String, commentAllLines: Bool = true) -> String {
+    var content = exampleYAMLContent(version: versionForDocLink)
+    if commentAllLines {
+      content = content
+        .split(separator: "\n", omittingEmptySubsequences: false)
+        .map { $0.isEmpty ? "" : $0.hasPrefix("#") ? "#\($0)" : "# \($0)" }
+        .joined(separator: "\n")
+    }
+    return content
+  }
+
+  // swiftlint:disable line_length function_body_length
+  private static func exampleYAMLContent(version: String) -> String {
+    """
+    # Note: all of the config entries below are just examples with placeholders. Be sure to edit and adjust to your needs when uncommenting.
+
+    # In case your config entries all use a common input/output parent directory, you can specify those here.
+    #   Every input/output paths in the rest of the config will then be expressed relative to these.
+    #   Those two top-level keys are optional and default to "." (the directory of the config file).
+    input_dir: MyLib/Sources/
+    output_dir: MyLib/Generated/
+
+
+    # Generate constants for your localized strings.
+    #   Be sure that SwiftGen only parses ONE locale (typically Base.lproj) – otherwise it will generate the same keys multiple times.
```

Comment:
I don't usually put strings files in the Base.lproj... Maybe change this to something like "typically en.lproj or Base.lproj, whichever is your development language"